ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -824,7 +808,7 @@ let on_notification server (notification : Client_notification.t) :
       let+ () = send_diagnostics server doc in
       state)
   | CancelRequest _ ->
-    log ~title:Logger.Title.Warning "ignoring cancellation";
+    Log.log ~section:"debug" (fun () -> Log.msg "ignoring cancellation" []);
```

Comment:
I believe that log function should be used to write a logTrace notification.
But then it's a different choice, it means that you'll have to read the logs in something like vscode, it won't be written in a file. The good side would be to leverage the LSP inspector, but right now it is down https://github.com/microsoft/language-server-protocol/issues/1089
It's also easier to read the logs in real time.

But then, writing to stderr or a file makes sure that the info will be displayed even if the connection between the server and the editor is somehow broken

Reply:
I use Emacs, and I can already see the stderr for the server. Is this not possible for vscode?